src/components/ui/confirm-finish-modal.tsx

```
@@ -0,0 +1,37 @@
+      role="dialog"
+      aria-modal="true"
+    >
+      <div className="bg-white rounded-2xl border border-brand-primary-950 p-6 max-w-sm w-full shadow-lg relative">
+        <h2 className="text-lg font-bold mb-2">Leaving so soon?</h2>
+        <p className="text-sm text-gray-600 mb-4">Are you sure you want to finish the quiz?</p>
+        <div className="flex justify-end gap-3">
+          <Button variant="outline" onClick={onClose}>
+            No.
+          </Button>
+          <Button variant="default" onClick={onConfirm}>
+            Yes, I'm done! 🤥
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)